web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v3110/UpdateMetadataStatus.java

```diff
@@ -115,9 +118,14 @@ private void addMissingColumn(final Connection connection, Dialect dialect ) thr
         } catch (Exception e) {
             // If there was an error then we will log the error and continue.
             // Most likely cause is that the column already exists which should be fine.
-            Log.error(Geonet.DB, "  Exception while adding new " + MetadataStatus_.id.getName() + " column to " + MetadataStatus.TABLE_NAME + ". " +
-                    "Error is: " + e.getMessage());
-            Log.debug(Geonet.DB, e);
+            LOGGER.error(
+                Geonet.DB_MARKER,
+                "  Exception while adding new {} column to {}. Error is: ",
```

Comment:
Seem missing a parameter `{}` for `e.getMessage()`

Reply:
trailing attributes do not require ``{}`` but we can add for clarity
```suggestion
                "  Exception while adding new {} column to {}. Error is: {}",
```